ui/package.json

```diff
@@ -1,6 +1,16 @@
 {
   "name": "Bassa",
   "version": "1.3.0",
+  "description": "Automated Download Queue for Enterprise to take the best use of Internet bandwidth",
+  "bugs": "https://github.com/scorelab/Bassa/issues",
+  "author": "Malithsen",
+  "contributors": "Malithsen",
```

Comment:
This should contain all the contributors mentioned [here](https://github.com/scorelab/Bassa/graphs/contributors) separated by commas

Reply:
@malithsen sure :)